presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroup.java

```diff
@@ -941,9 +987,12 @@ public int hashCode()
     {
         private AtomicBoolean taskLimitExceeded = new AtomicBoolean();
 
-        public RootInternalResourceGroup(String name, BiConsumer<InternalResourceGroup, Boolean> jmxExportListener, Executor executor)
+        public RootInternalResourceGroup(
+                String name,
+                BiConsumer<InternalResourceGroup, Boolean> jmxExportListener,
+                Executor executor)
         {
-            super(Optional.empty(), name, jmxExportListener, executor);
+            super(Optional.empty(), name, jmxExportListener, executor, true);
```

Comment:
Why is this `true`? Sorry I don't have much background about resource group.

Reply:
I guess the root resource group is always static

Reply:
Yes!